.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,18 @@
+name: Feature request
+description: Provide idea for introducing a new feature
+labels:
+- type/feature
+body:
+- type: textarea
+  id: feature
+  attributes:
+    label: What would you like to be added?
+  validations:
+    required: true
+
+- type: textarea
+  id: rationale
+  attributes:
+    label: Why is this needed?
+  validations:
+    required: true
```

Comment:
Nit, please address GH formatting need `no newline at end of file` (here and other new templates added)